### ISSUE TITLE
nf-core subworkflows create-test-yml can not find files in /tmp/*

### ISSUE THREAD

**Issue by mahesh-panchal** [nf-core/tools#2003]
### Description of the bug

`nf-core subworkflows create-test-yml` can not find the output files in a `/tmp/` folders.

### Command used and terminal output

```console
$ pip install --upgrade --force-reinstall git+https://github.com/nf-core/tools.git@dev
Collecting git+https://github.com/nf-core/tools.git@dev
  Cloning https://github.com/nf-core/tools.git (to revision dev) to /tmp/pip-req-build-zr1g8ig3
  Running command git clone --filter=blob:none --quiet https://github.com/nf-core/tools.git /tmp/pip-req-build-zr1g8ig3
  Running command git checkout -b dev --track origin/dev
  Switched to a new branch 'dev'
  branch 'dev' set up to track 'origin/dev'.
  Resolved https://github.com/nf-core/tools.git to commit 1e36a4fe6a860147e68c5dc90e5afc9ca3042402
...
$ nf-core subworkflows create-test-yml fastq_fastqc_umitools_trimgalore -f -p

                                          ,--./,-.
          ___     __   __   __   ___     /,-._.--~\
    |\ | |__  __ /  ` /  \ |__) |__         }  {
    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                          `._,._,'

    nf-core/tools version 2.7.dev0 - https://nf-co.re


INFO     Looking for test workflow entry points: 'tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore/main.nf'                                                                                                                                                                         test_yml_builder.py:130
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
INFO     Building test meta for entry point 'test_fastq_fastqc_umitools_trimgalore_single'                                                                                                                                                                                                     test_yml_builder.py:164
INFO     Setting env var '$PROFILE' to an empty string as not set.                                                                                                                                                                                                                             test_yml_builder.py:325
         Tests will run with Docker by default. To use Singularity set 'export PROFILE=singularity' in your shell before running this command.                                                                                                                                                                        
INFO     Running 'fastq_fastqc_umitools_trimgalore' test with command:                                                                                                                                                                                                                         test_yml_builder.py:349
         nextflow run ./tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore -entry test_fastq_fastqc_umitools_trimgalore_single -c ./tests/config/nextflow.config --outdir /tmp/tmp6kw2yur3 -work-dir /tmp/tmpyah9z85l                                                                                        
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Repeating test ...                                                                                                                                                                                                                                                                    test_yml_builder.py:352
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Test workflow finished!                                                                                                                                                                                                                                                               test_yml_builder.py:366
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
INFO     Building test meta for entry point 'test_fastq_fastqc_umitools_trimgalore_paired'                                                                                                                                                                                                     test_yml_builder.py:164
INFO     Running 'fastq_fastqc_umitools_trimgalore' test with command:                                                                                                                                                                                                                         test_yml_builder.py:349
         nextflow run ./tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore -entry test_fastq_fastqc_umitools_trimgalore_paired -c ./tests/config/nextflow.config --outdir /tmp/tmp9p7o4_1q -work-dir /tmp/tmp4b8v0yxg                                                                                        
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Repeating test ...                                                                                                                                                                                                                                                                    test_yml_builder.py:352
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Test workflow finished!                                                                                                                                                                                                                                                               test_yml_builder.py:366
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
INFO     Building test meta for entry point 'test_fastq_fastqc_umitools_trimgalore_'                                                                                                                                                                                                           test_yml_builder.py:164
INFO     Running 'fastq_fastqc_umitools_trimgalore' test with command:                                                                                                                                                                                                                         test_yml_builder.py:349
         nextflow run ./tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore -entry test_fastq_fastqc_umitools_trimgalore_ -c ./tests/config/nextflow.config --outdir /tmp/tmpg17a_r_e -work-dir /tmp/tmp65dd66cf                                                                                              
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Repeating test ...                                                                                                                                                                                                                                                                    test_yml_builder.py:352
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Test workflow finished!                                                                                                                                                                                                                                                               test_yml_builder.py:366
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
INFO     Building test meta for entry point 'test_fastq_fastqc_umitools_trimgalore_umi'                                                                                                                                                                                                        test_yml_builder.py:164
INFO     Running 'fastq_fastqc_umitools_trimgalore' test with command:                                                                                                                                                                                                                         test_yml_builder.py:349
         nextflow run ./tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore -entry test_fastq_fastqc_umitools_trimgalore_umi -c ./tests/config/nextflow.config --outdir /tmp/tmp_iwdupaj -work-dir /tmp/tmpsl4j0urn                                                                                           
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Repeating test ...                                                                                                                                                                                                                                                                    test_yml_builder.py:352
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Test workflow finished!                                                                                                                                                                                                                                                               test_yml_builder.py:366
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
INFO     Building test meta for entry point 'test_fastq_fastqc_umitools_trimgalore_skip'                                                                                                                                                                                                       test_yml_builder.py:164
INFO     Running 'fastq_fastqc_umitools_trimgalore' test with command:                                                                                                                                                                                                                         test_yml_builder.py:349
         nextflow run ./tests/subworkflows/nf-core/fastq_fastqc_umitools_trimgalore -entry test_fastq_fastqc_umitools_trimgalore_skip -c ./tests/config/nextflow.config --outdir /tmp/tmpqoactqzh -work-dir /tmp/tmpy2jeqmvo                                                                                          
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Repeating test ...                                                                                                                                                                                                                                                                    test_yml_builder.py:352
Picked up JAVA_TOOL_OPTIONS:  -Xmx3489m
INFO     Test workflow finished!                                                                                                                                                                                                                                                               test_yml_builder.py:366
CRITICAL Could not find any test result files in '/tmp/tmpqoactqzh'
```


### System information

* Nextflow: 22.04.0 build 5697
* Hardware: Cloud - Gitpod
* Executor: local, docker
* OS: unix - Gitpod
* nf-core/tools: `2.7.dev0` commit: 1e36a4fe6a860147e68c5dc90e5afc9ca3042402
* python: Python 3.9.13

**Comment by mirpedrol** [comment]
Hello @mahesh-panchal 
I have been trying this and I thing there's something wrong with one of the tests `test_fastq_fastqc_umitools_trimgalore_skip`. The command works for the other tests and also for other subworkflows.
I have tried to remove that test, and then it works. 
In case it is useful for debugging, I have also tried to run the tests manually with `TMPDIR=~ PROFILE=conda pytest --tag fastqc --symlink --kwd --git-aware --color=yes` and the work directory for that test is empty.
